.eslintcache

```diff
@@ -1 +1 @@
gers/src/index.js",[],["61","62"],"/Users/spideyfandan/ga/hackathon/type-tigers/src/pages/App/App.jsx",[],"/Users/spideyfandan/ga/hackathon/type-tigers/src/utilities/users-service.js",[],"/Users/spideyfandan/ga/hackathon/type-tigers/src/pages/AuthPage/AuthPage.jsx",[],"/Users/spideyfandan/ga/hackathon/type-tigers/src/pages/OrderHistoryPage/OrderHistoryPage.jsx",[],"/Users/spideyfandan/ga/hackathon/type-tigers/src/pages/NewOrderPage/NewOrderPage.jsx",[],"/Users/spideyfandan/ga/hackathon/type-tigers/src/components/NavBar/NavBar.jsx",[],"/Users/spideyfandan/ga/hackathon/type-tigers/src/utilities/users-api.js",[],"/Users/spideyfandan/ga/hackathon/type-tigers/src/pages/Game/index.jsx",[],"/Users/spideyfandan/ga/hackathon/type-tigers/src/components/LoginForm/LoginForm.jsx",[],"/Users/spideyfandan/ga/hackathon/type-tigers/src/components/SignUpForm/SignUpForm.jsx",[],"/Users/spideyfandan/ga/hackathon/type-tigers/src/components/GameWord/index.jsx",["63","64"],"import React, {useState, useEffect} from 'react';\n\nexport default function GameWord({word, setWord, testArr}) {\n\n  // Display word\n  // Make word fade\n  // Handle submit\n  // Handle change\n  \n\n\n  useEffect(()=>{\n    setWord(testArr[0])\n    const timer = setTimeout(() => {\n        setWord('');\n    }, 1000);\n    return () => clearTimeout(timer);\n  }, [])\n\n\n  return(\n    <>\n      <p>{word}</p>\n      \n    </>\n  )\n}","/Users/spideyfandan/ga/hackathon/type-tigers/src/components/Answer/index.jsx",["65"],"import {useState} from 'react';\nimport Letters from \"../Letters/index\"\n\nexport default function Answer({answer}) {\n\n\n  \n  function handleChange(e) {\n    console.log(e);\n  }\n  \n  return(\n    <>\n    <Letters />\n      <form>\n        <input onChange={handleChange} type=\"text\" name=\"word\" id=\"\" value={answer}/>\n      </form>\n    </>\n  )\n}","/Users/spideyfandan/ga/hackathon/type-tigers/src/components/Letters/index.jsx",[],{"ruleId":"66","replacedBy":"67"},{"ruleId":"68","replacedBy":"69"},{"ruleId":"70","severity":1,"message":"71","line":1,"column":16,"nodeType":"72","messageId":"73","endLine":1,"endColumn":24},{"ruleId":"74","severity":1,"message":"75","line":18,"column":6,"nodeType":"76","endLine":18,"endColumn":8,"suggestions":"77"},{"ruleId":"70","severity":1,"message":"71","line":1,"column":9,"nodeType":"72","messageId":"73","endLine":1,"endColumn":17},"no-native-reassign",["78"],"no-negated-in-lhs",["79"],"no-unused-vars","'useState' is defined but never used.","Identifier","unusedVar","react-hooks/exhaustive-deps","React Hook useEffect has missing dependencies: 'setWord' and 'testArr'. Either include them or remove the dependency array. If 'setWord' changes too often, find the parent component that defines it and wrap that definition in useCallback.","ArrayExpression",["80"],"no-global-assign","no-unsafe-negation",{"desc":"81","fix":"82"},"Update the dependencies array to be: [setWord, testArr]",{"range":"83","text":"84"},[351,353],"[setWord, testArr]"]
```

Comment:
Same here. You may have to `git rm --cached .eslintcache` to git .gitignore to properly ignore both this and then do the same to .env file.